src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/RemoveAzureStorageAccount.cs

```diff
@@ -44,6 +44,8 @@ public override void ExecuteCmdlet()
         {
             base.ExecuteCmdlet();
 
+            WriteWarning("Will add confirmation of executing the cmdlet and Force parameter in a future release. ");
```

Comment:
Rather than writing a warnign like this, we should attribute the obsoleted parameter.  Please hold off until the ShouldProcess PR comes in.